docs/csharp/programming-guide/linq-query-expressions/toc.md

```diff
@@ -1,19 +0,0 @@
-# [LINQ Query Expressions](index.md)
```

Comment:
@mairaw Thanks for identifying the error. I removed the entry from the TOC, and made sure that there weren't others that were in my list of deleted files for this PR.